docs/memory-safety-continuum.md

```diff
@@ -16,16 +16,16 @@ Rather than using terms like "Memory Safe Language" and "Memory Unsafe Language"
 
 ## The Continuum - Understanding Where Your Software Is Now
 
-This is a very rough idea of what the continuum might look like - from "least safety" to "most safety"
+The continuum we proose, from "least safety" to "most safety", is a way to understand where your software is now in terms of memory safety and what you can do to improve it. The ambition of developers and organizations should be to follow **current** best practice for the eco system that they are in, and to use the modern tool chains for that eco system towards a more memory safe state.
 
 * Using a non-memory safe by default language (such as C or C++) without developer best practices or automated tooling to check for memory safety
 * Using a non-memory safe by default language with developer best practices, but no automated tooling to check for memory safety
-* Using a non-memory safe by default language with developer best practices and automated tooling to check for memory safety in first party code
-* Using a non-memory safe by default language with developer best practices and automated tooling to check for memory safety in first party code AND automated tooling to check for memory safety in third party code (dependencies)
-* Using a memory safe by default language (such as Rust, Go, Python, Java, JavaScript, C#) without developer best practices and automated tooling
-* Using a memory safe by default language with developer best practices, but no automated tooling to check for memory safety
-* Using a memory safe by default language with developer best practices and automated tooling to check for memory safety in first party code
-* Using a memory safe by default language with developer best practices and automated tooling to check for memory safety in first party code AND automated tooling to check for memory safety in third party code (dependencies)
+* Using a non-memory safe by default language with developer best practices and automated tooling to check for memory safety in your software's code
+* Using a non-memory safe by default language with developer best practices and automated tooling to check for memory safety in your software's code as well as in all your software's dependencies code
+* Using a memory safe by default language (such as Rust, Go, Python, Java, JavaScript, C#) without developer best practices and compiler-activated plugin, post processors etc. and without additional automated tooling that checks for memory safety 
```

Comment:
Lists of things should always be in alpha order unless the specific order is described.

Reply:
This wording is better. I still find it quite academic.

If I write code in a programming language, I want to do know if the default toolchain in its default configuration provides sufficient safety. If so, I'm done, unless I am writing unsafe code.

The problem with this continuum is that line 25 is the first line describing safe languages and it doesn't strike me as indicative of a default configuration for C# or Rust, for example.

Two changes would help this a lot.

- Invert the order so that one doesn't need to work through the whole list to get to the good state.
- Provide examples. I'd love to see where a default Python install falls on this list, for example. Otherwise, this continuum is very hard to make concrete for the reader.